docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+        cost.rejected.operations:
+          type: histogram
+          value:
+            cost: estimated                                          # Estimated cost is used to populate the histogram
+          description: "Estimated cost per rejected operation."
+          unit: delta
+          condition:
+            eq:                                                      # Only show rejected operations.
+              - cost: result
+              - "COST_ESTIMATED_TOO_EXPENSIVE"
+          attributes:
+            graphql.operation.name: true                             # Graphql operation name is added as an attribute
+
+```
+
+This custom instrument may not be suitable for users where there are many operation names, such as a public internet-facing API. You can add conditions to reduce the number of returned operations. For example, use a condition that outputs results only when the cost delta is greater than a threshold:
+
+```yaml title="Example condition for cost delta threshold"
+telemetry:
+  instrumentation:
+    instruments:
+      supergraph:
+        # custom instrument
+        cost.rejected.operations:
+          condition:
+            all:        
+              - eq: # Only show rejected operations
+                 - cost: result
+                 - "COST_ESTIMATED_TOO_EXPENSIVE"
+#highlight-start
+              - gt: # Only show cost delta > 100
+                 - cost: delta
+                 - 100
+#highlight-end            
+```
+
+4. You should now be able to configure your APM to see which operations are too costly. Visualizing the histogram can be useful, such as with top-N or heatmap tools. 
+
+For example, the following table has the estimated cost of operations:
+
+| Operation name      | Estimated cost |
+|---------------------|----------------|
+| ExtractAll          | 9020           |
+| GetAllProducts      | 1435           |
+| GetLatestProducts   | 120            |
+| GetRecentlyUpdated  | 99             |
+| FindProductByName   | 87             |
+
+The `ExtractAll` operation has a very large estimated cost, so it's a good candidate to be rejected.
+
+Also, the value of the `cost.delta` metric—the difference between the actual and estimated cost—shows whether the assumed list size used for cost estimation is too large or small. In this example, the positive `cost.delta` means that the actual list size is greater than the estimated list size. Therefore the `static_estimated.list_size` can be reduced to closer match the actual. 
+
+### Improve cost estimation model
+
+You should iteratively improve your cost estimation model. Accurate cost estimation is critical to identifying and preventing queries that could harm your subgraphs.
+
+1. The previous step identified a noticeable difference between actual and estimated costs with the example operations. You can better understand the difference—and consequently tune the configured list size—by adding telemetry instruments for fields in your GraphQL schema.
```

Comment:
Since there is now step 2, 3, etc. adding a `1.` here feels strange.

```suggestion
The previous step identified a noticeable difference between actual and estimated costs with the example operations. You can better understand the difference—and consequently tune the configured list size—by adding telemetry instruments for fields in your GraphQL schema.
```